applications/plugins/SofaPython/Binding_Simulation.cpp

```diff
@@ -0,0 +1,126 @@
+/******************************************************************************
+*       SOFA, Simulation Open-Framework Architecture, development version     *
+*                (c) 2006-2017 INRIA, USTL, UJF, CNRS, MGH                    *
+*                                                                             *
+* This program is free software; you can redistribute it and/or modify it     *
+* under the terms of the GNU Lesser General Public License as published by    *
+* the Free Software Foundation; either version 2.1 of the License, or (at     *
+* your option) any later version.                                             *
+*                                                                             *
+* This program is distributed in the hope that it will be useful, but WITHOUT *
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License *
+* for more details.                                                           *
+*                                                                             *
+* You should have received a copy of the GNU Lesser General Public License    *
+* along with this program. If not, see <http://www.gnu.org/licenses/>.        *
+*******************************************************************************
+* Authors: The SOFA Team and external contributors (see Authors.txt)          *
+*                                                                             *
+* Contact information: contact@sofa-framework.org                             *
+******************************************************************************/
+
+#include "PythonFactory.h"
+#include "Binding_Simulation.h"
+#include "Binding_Base.h"
+#include "PythonToSofa.inl"
+#include <sofa/simulation/Simulation.h>
+
+static inline sofa::simulation::Simulation* get_simulation(PyObject* obj) {
```

Comment:
The Simulation class is also very weird in Sofa's code base as it point to some static singleton like object. 

It is unclear to me if the binding of theses features shouldn't go into a the same module as it was proposed to GUIManager (eg: SofaApplication / SofaRunTime). 

Eg:
```python
SofaApplication.setSimulation()
SofaApplication.unload()
```